nomad/structs/structs.go

```diff
@@ -5202,7 +5202,7 @@ type TaskGroup struct {
 
 	// ShutdownDelay is the amount of time to wait between deregistering
 	// group services in consul and stopping tasks.
-	ShutdownDelay *time.Duration
+	ShutdownDelay time.Duration
```

Comment:
TaskGroup is persisted structure - the type change might affect backward compatibility - need to test how it handles upgrades.

Reply:
It's also transmitted over RPC between agents, so we'd need to test with mismatched agent versions as well.

Probably safest just to revert this part of the change.